packages/util/src/api.ts

```diff
@@ -1569,6 +1569,76 @@ export class Api extends BaseApi {
       : secondAmountNum.div(!firstAmountNum.isZero() ? firstAmountNum : one)
     return result.format()
   }
+
+  /**
+   * Returns the referral of the invited user by Id
+   * @param invitedUserId address of invited account
+   * @returns referral
+   */
+  public async getReferral (invitedUserId: string): Promise<string> {
+    const referral = await this.api.query.referrals.referrers(invitedUserId) as any
+    return !referral ? '' : referral.toString()
+  }
+
+  /**
+   * Returns invited users of the referral
+   * @param referralId address of referral account
+   * @returns array of invited users
+   */
+  public async getInvitedUsers (referralId: string): Promise<Array<string>> {
+    const referrals = await this.api.query.referrals.referrals(referralId) as any
+    if (!referrals) {
```

Comment:
Looks like if account hasn't referrals, this method already returns empty array (checked it in polkadotjs apps). Or another falsy values could be returned by `api.query.referrals.referrals`?

Reply:
Yes, you are right, fixed.